osgeo_importer/templates/osgeo_importer/new.html

```diff
@@ -48,9 +48,6 @@ <h2 class="page-title">{% trans "Add data" %}</h2>
                 <i class="fa fa-warning">  {{ error|escape }}</i>
             {% endfor %}
             </div>
-            <div class="col-md-6" style="padding-top: 5px; padding-left: 2px;">
```

Comment:
See my comment about this in the PR description -- essentially 1) removing it puts us in the same state as current (non-osgeo) importer, so it's not truly a UX regression; 2) yep we should be generating it dynamically based on values defined elsewhere to avoid stale markup (ref: https://github.com/boundlessgeo/django-osgeo-importer/issues/19)